docs/adapt-theme-to-brand.md

```diff
@@ -3,4 +3,89 @@ id: adapt-theme-to-brand
 title: Adapt the theme to your brand
 ---
 
-Adapt the theme to your brand
\ No newline at end of file
+When you start a new project, one of the first things that you will need to do
+is to adapt the look and feel of Front Commerce to match your brand's style.
+
+Obviously, it is not something that can be done in just a few minutes because
+your identity shines in the details. However, you can do all the heavylifting
+pretty quickly by customizing what is called **Design Tokens**.
+
+## Get your Design Tokens
+
+This concept is often closely related to Design Systems. Basically Design Tokens
+are the _core styles_ of your design: colors, font families, font sizes,
+borders, shadows, etc. Together they are what make your brand unique.
+
+There even exist tools that extract those tokens from existing websites. In this
+example, we will use [CSS Stats](https://cssstats.com/) to extract our Design
+Tokens.
+
+https://cssstats.com/stats?url=https%3A%2F%2Fwww.smashingmagazine.com&ua=Browser%20Default
+Example with Smashing Magazine's website but feel free to use your brand's
+website
+
+If you want to learn more about it, you can have a look at
+[Design tokens for dummies](https://uxdesign.cc/design-tokens-for-dummies-8acebf010d71)
+which is a very nice introduction.
+
+## Apply these tokens to your theme
+
+Now that we've got our Design Tokens, let's apply them to Front Commerce's
+theme.
+
+Since we use the Atomic Design principles, the tokens are within the atoms in
+`/web/theme/ui/atoms`. In this guide, we'll focus on the colors and the
+typography settings. But feel free to go farther and edit buttons, form inputs,
```

Comment:
Further is more adapted here: https://en.oxforddictionaries.com/usage/farther-or-further.